internal/consensus/state.go

```diff
@@ -917,7 +917,6 @@ func (cs *State) handleMsg(mi msgInfo) {
 			"round", cs.Round,
 			"peer", peerID,
```

Comment:
```suggestion
			"peer", peerID,
			"type", fmt.Sprintf("%T", msg),
```
Perhaps it's helpful to add the type of message?